lib/ProductOpener/APITest.pm

```diff
@@ -101,8 +101,13 @@ sub wait_auth() {
 	my $count = 0;
 	my $ua = new_client();
 	my $target_url = construct_test_url("");
+	my $discovery_endpoint
```

Comment:
IIRC, at least in my first tests, we needed the public URL to be used for the discovery endpoint, so that the URL could be used for redirects. Otherwise, the backchannel URL (internal docker net) was used for public URLs, and thus not really working from the internet @john-gom

Reply:
I checked the response and it seemed to be returning the public URL for those kinds of thing, but will double check